rfc/0054-sdk3-management-apis.md

```diff
@@ -582,6 +666,37 @@ void BuildDeferredIndexes(string bucketName, [options])
 
   * `Timeout` or `timeoutMillis` (`int`/`duration`) - the time allowed for the operation to be terminated. This is controlled by the client.
 
+  * `CollectionName` - the name of the collection to restrict indexes to.
+  * `ScopeName` - the name of the scope to restrict indexes to.
+    * If either `CollectionName` or `ScopeName` are set then both *must* be set.
+
+### N1QL
+
+No collection set, runs build index on only deferred indexes created on the bucket, this does not include indexes created on scopes and collections:
+```
+
+"BUILD INDEX ON `bucketName` (
+  (
+    SELECT RAW name FROM system:indexes 
+    WHERE (keyspace_id = "bucketName" AND bucket_id IS MISSING)
```

Comment:
nit: `( )` are not needed now anymore

Reply:
I do not remember if this was mentioned previously, but it looks like this will not work on server versions < 6.5.  I see a syntax error when trying to use a sub-query to serve as the list of indexes to build. :(

`BUILD INDEX ON `default` ((
	SELECT RAW idx.name FROM system:indexes AS idx
	WHERE (
		(`bucket_id` IS MISSING AND `keyspace_id`="default")
		OR `bucket_id`="default"
		) AND state="deferred"
	))
`